python/lsst/ts/adamSensors/adamSensorsCSC.py

```diff
@@ -26,23 +34,98 @@ def __init__(self, config_dir=None, initial_state=salobj.State.STANDBY, initial_
     async def begin_start(self, id_data):
```

Comment:
Seems like this start command can take a while to execute. If it takes longer than 1 sec (1 heartbeat) you need to send an `Ack.INPROGRESS` at the start. Add the following:

```
        self.cmd_start.ack_in_progress(
            id_data, timeout=self.start_timeout
        )
```

where `self.start_timeout` is an estimated of how long this command will take to execute.

Also, `id_data` is the old name of the parameter. You should probably rename it to `data` only (https://github.com/lsst-ts/ts_salobj/blob/95a7cc70d17e15533ee9e24b13aec28e89928206/python/lsst/ts/salobj/base_csc.py#L567).